api/config_entry_gateways_test.go

```diff
@@ -0,0 +1,127 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPI_ConfigEntries_IngressGateway(t *testing.T) {
```

Comment:
FYI, here's another test to update: `TestParseConfigEntry`
https://github.com/hashicorp/consul/pull/7532#issuecomment-605298562

Reply:
Thanks, added a test!